client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -58,38 +74,93 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 	}
 
 	onScriptLoaded = ( error ) => {
+		const {
+			isPlaying,
+			item,
+			onScriptLoadError,
+		} = this.props;
+
 		if ( error ) {
-			log( `Script${ error.src } failed to load.` );
+			log( `Script${ get( error, 'src', '' ) } failed to load.` );
+			onScriptLoadError();
+
 			return;
 		}
 
 		const {
-			isPlaying,
-			item,
-		} = this.props;
+			height = 480,
+			videopress_guid,
+			width = 854,
+		} = pick( item, [ 'videopress_guid', 'height', 'width' ] );
```

Comment:
why are we using `pick()` here? the destructuring assignment should already only grab what we need - compared to `const { height, … } = item` what is the advantage?